### ISSUE TITLE
Raises warnings in Django 1.8 and will fail with Django 1.9

### ISSUE THREAD

**Issue by jribbens** [divio/djangocms-inherit#14]
```forms.py``` says:
```python
from django.forms.util import ErrorList
```
This raises `RemovedInDjango19Warning` with current Django and will fail in future releases. It should be:
```python
from django.forms.utils import ErrorList
```

**Comment by mkoistinen** [comment]
Thanks for this. However, this will fail in Django 1.6, which is currently still supported.

**Comment by mkoistinen** [comment]
Actually, there is already a PR open for this specific change. #13.

**Comment by jribbens** [comment]
Weird, that PR doesn't show up in github's list of issues, nor has its patch been applied.

**Comment by mkoistinen** [comment]
Closed by #13, #16.
